docs/DeveloperGuide.md

```diff
@@ -173,6 +173,32 @@ The rake symbol (in the Check validity of command action above) is used to show
 
 ### Implementation of Main Mode Features
 
+List of decks and list of cards are stored separately in `UniqueDeckList` and `UniqueCardList`. Although a card belongs to a deck, it is not stored within a deck. Each card indicates a deck that it belongs to itself.
+A deck stores its name and a Boolean value of whether it is selected or not.
```

Comment:
I'm not sure if this is important. I think this attribute is only used in the FXML. Please correct me if I'm wrong @rockman007372 @junianob

Reply:
Maybe not needed here. Also I addressed this in my newest PR, which has implementations for MasterDeck.